_posts/2016-01-06-format-support.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Supporting complex formats - what we will and won't do, and what you can do to help
+categories: file-formats community
+---
+
+You may have noticed that a few months ago, we received an email asking us
+about when we expect to support 3D HISTECH .mrxs files. This sort of request
+isn’t particularly unusual and the reply gives an insight into one of the key 
+challenges we face.
+
+##Just because we don’t have a reader, doesn’t mean we haven’t done any work
+
+3D HISTECH .mrxs is an example of a complex format, the design of which does
+not make our work any easier. In fact, we can say with some confidence that
+the 3D HISTECH .mrxs file format is the most complex whole slide imaging file
+format we have ever encountered. We can say this because although we haven’t
+delivered a full reader for .mrxs—and there hasn’t been substantial public
+development—we have spent a great deal of time examining the format and
+potential solutions, and building test readers. Thanks to the example data the
+community has generously provided, we have been able to analyse the on-disk
+layout as well as the compression types and map out the details of what an
+implementation would entail.
+
+Unfortunately, the result of all this work has been the conclusion that we
+simply do not have the resources to prioritize delivering a complete solution
+for this format. This is not the only format we have reached this conclusion
+about. For example, support for [3i Slidebook 6 files](http://www.openmicroscopy.org/site/support/bio-formats/formats/3i-slidebook.html) was only added to
+Bio-Formats last May when [3i](https://www.intelligent-imaging.com) committed
+to developing the reader themselves.
+Obviously, we are very grateful for this, but that doesn’t change the fact
+that we had already spent years working on various versions of this format
+(our initial single-series Slidebook reader was released back in 2006 and
+obviously the work to produce it started even further back than that). Nikon
+ND2 and Zeiss CZI are other examples of formats with a complex design that has
```

Comment:
I'm not sure if this should be 'has' or 'have', the following `them` implies it is the examples that are the subject of the verb.